tool/templates/js2/method.js.jinja

```diff
@@ -5,7 +5,7 @@
     {%- if typescript %}: {{ param.ty -}}{%- endif -%}
     {%- if !loop.last -%}, {% endif -%}
 {%- endfor -%}
-) {%- if typescript %}{{ return_type }};{% else %} {
+) {%- if typescript %}{% if return_type != ": void" %}{{ return_type }}{% endif %};{% else %} {
```

Comment:
Actually I don't think this is correct. When I apply this, I get the warning

```
<method> implicitly has an 'any' return type, but a better type may be inferred from usage. ts(7050)
```

However, this *does* seem to be required for setters, where without I get the error

```
A 'set' accessor cannot have a return type annotation. ts(1095)
```

Reply:
done